sps20/content/aboutus/contents.lr

```diff
@@ -0,0 +1,31 @@
+title: About us
+---
+description: Our history and other facts
+---
+ordering: 8
+---
+body:
+
+# About us
+
+## Non-profit
+
+We are a non-profit volunteer organization funded entirely by our participants, donators and sponsors. The conference is organized by the Swiss Python Summit Association.
+
+We are always looking for people to [help us organize the conference](/helpus) or [support us with a donation](/helpus#donate).
+
+## Code of Conduct
+
+Our conference is dedicated to providing a harassment-free conference experience for everyone, regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion (or lack thereof), or technology choices. We do not tolerate harassment of conference participants in any form. Sexual language and imagery is not appropriate for any conference venue, including talks, workshops, parties, Twitter and other online media. Conference participants violating these rules may be sanctioned or expelled from the conference without a refund at the discretion of the conference organisers.
+
+See [the full Code of Conduct](/coc).
+
+## History
+
+After the good experience of [EuroPython 2015](https://ep2015.europython.eu/), a small group of enthusiasts came up with the idea to organise a Python conference in Switzerland. A few meetings and discussions later, we organized a small and very inexpensive event in early 2016. We expected about 50 people, but 130 signed up. It was a great success, so we decided to continue organising the conference. Two more editions of the conference were held in 2017 and 2018, and after a short break, another one in 2022. After some new members and a new board were found, the Swiss Python Summit was continued with annual summits from 2023. We look forward to seeing you at the next summit!

```

Comment:
```suggestion
After the good experience of [EuroPython 2015](https://ep2015.europython.eu/), a small group of enthusiasts came up with the idea to organize a Python conference in Switzerland. A few meetings and discussions later, we organized a small and very inexpensive event in early 2016. We expected about 50 people, but 130 signed up. It was a great success, so we decided to continue organizing the conference. Two more editions of the conference were held in 2017 and 2018, and after a short break, another one in 2022. After some new members and a new board were found, the Swiss Python Summit was continued with annual summits from 2023. We look forward to seeing you at the next summit!
```

Reply:
I thought we agreed on BE. but personally i don't care if BE or AE and as just found out it's a wild mix now..